Flow.Launcher.Core/Plugin/JsonRPCPluginV2.cs

```diff
@@ -139,11 +139,18 @@ public virtual Task ReloadDataAsync()
             return Task.CompletedTask;
         }
 
-        public virtual ValueTask DisposeAsync()
+        public virtual async ValueTask DisposeAsync()
         {
+            try
+            {
+                await RPC.InvokeAsync("close");
+            }
+            catch (RemoteMethodNotFoundException e)
+            {
+            }
+
             RPC?.Dispose();
             ErrorStream?.Dispose();
```

Comment:
Actually, sorry, one more thing: shouldn't .Dispose() here be in a `finally` block now, so they're disposed of regardless of if there are any other exceptions or not?